config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   root 'welcome#index'
+  get '/quotes', to: 'welcome#quotes'
```

Comment:
Could this be taken to it's own page? Essentially you are creating a quotes index page. Can you explain why you made the design choice to not be RESTful by making it an action in the welcome controller?